### PR TITLE
docs: how to get help for sub-sub-commands

### LINKED ISSUE #6232: borg help does not show help for sub-sub-commands (eg. borg key export)
## Have you checked borgbackup docs, FAQ, and open Github issues?

Yes

## Is this a BUG / ISSUE report or a QUESTION?

ISSUE

#### Your borg version (borg -V).

borg 1.2.0b5.dev34+g91cfa823

#### Operating system (distribution) and version.

Fedora release 34 (Thirty Four)

#### Full borg commandline that lead to the problem (leave away excludes and passwords)

```sh
borg help key export
usage: borg [-V] [-h] [--critical] [--error] [--warning] [--info] [--debug]
            [--debug-topic TOPIC] [-p] [--iec] [--log-json]
            [--lock-wait SECONDS] [--bypass-lock] [--show-version] [--show-rc]
            [--umask M] [--remote-path PATH] [--remote-ratelimit RATE]
            [--upload-ratelimit RATE] [--remote-buffer UPLOAD_BUFFER]
            [--upload-buffer UPLOAD_BUFFER] [--consider-part-files]
            [--debug-profile FILE] [--rsh RSH]
            <command> ...
borg: error: unrecognized arguments: export
```

## Describe the problem you're observing.

`borg help` appears to have no support for sub-sub-commands such as `borg key export`

#### Can you reproduce the problem? If so, describe how. If not, describe troubleshooting steps you took before opening the issue.

Run `borg help key export`

## A workaround

This works:
```sh
borg key export --help
```

### PR DESCRIPTION
**Pull request by ReethuVinta**
Fixes #6232

### CODE REVIEW

**Comment by ReethuVinta** [comment]
For #6278

File: docs/usage/general.rst [hunk 1]
```diff
@@ -45,6 +45,12 @@ that is the safer way; see the warning below.
     experience errors and read broken data if changes to that repository are
     being made at the same time.
 
+Option ``--help`` when used as a command for help on subcommands(eg. ``borg help compact``), 
+works as expected. But, it does not work when used on sub-sub-commands(eg. ``borg help key export``).
+There is a workaround to get help on sub-sub-commands using the help command as a flag(eg. 
+``borg key export --help``).
```
Comment (ThomasWaldmann) [inline docs/usage/general.rst:51]: Please proofread this again.

Comment (ReethuVinta) [reply docs/usage/general.rst]: Done. Can you please confirm and close this issue?

**Comment by ThomasWaldmann** [comment]
guess you need to rebase this onto a current master after the other PR is merged.

File: docs/usage/general.rst [hunk 2]
```diff
@@ -45,6 +45,10 @@ that is the safer way; see the warning below.
     experience errors and read broken data if changes to that repository are
     being made at the same time.
 
+Option ``--help`` when used as a command works as expected on subcommands(e.g., ``borg help compact``).
+But it does not work when the help command is being used on sub-sub-commands(e.g., ``borg help key export``).
+The workaround for this to use the help command as a flag(e.g., ``borg key export --help``).
```
Comment (ThomasWaldmann) [inline docs/usage/general.rst:50]: guess there is a space missing in front of each of the 3 opening parentheses.

Comment (ReethuVinta) [reply docs/usage/general.rst]: added it

**Comment by ThomasWaldmann** [comment]
guess you need to rebase this onto a current master.

please put everything into 1 commit with a good commit comment.

**Comment by ThomasWaldmann** [comment]
Thanks!

**Pull request merged**